doc/en/user/source/styling/sld/language.rst

```diff
@@ -52,4 +54,4 @@ Here is an example providing labels in multiple languages, integrating the ``lan
          </TextSymbolizer>
 
 
-If the Function has been defined in a style and the ``LANGUAGE`` parameter has not been specified in the request or has value ``LANGUAGE=*`` the default language will be used. See :ref:`internationalization` for details on ``Default Language``.
+If the Function has been defined in a style and the ``LANGUAGE`` parameter has not been specified in the request or has value ``LANGUAGE=*`` the default value will be returned. If there is no default value, default language will be returned. See :ref:`internationalization` for details on ``Default Language``.
```

Comment:
I would put it differently telling that from the example it is possible to see that by putting an empty property name it is possible to match a value with a notExisting/* language parameter.

Reply:
Done